src/command_line_parser.cc

```diff
@@ -490,6 +491,16 @@ CLParser::Usage(const std::string& msg)
                    "which means that no warmup requests will be sent.",
                    18)
             << std::endl;
+  std::cerr
+      << FormatMessage(
+             " --fixed-schedule: Enables fixed schedule inference load mode. "
+             "In this mode, Perf Analyzer runs through the `--input-data` JSON "
+             "once. Each entry represents a request and must have a "
+             "`timestamp` \"input\", which tells Perf Analyzer when to send "
+             "that request. A timestamp of `N` means Perf Analyzer will send "
+             "that request `N` milliseconds after the start of the benchmark.",
+             18)
+      << std::endl;
```

Comment:
Again, PA was missing help option docs for fixed schedule.

Adding in this PR.